Resources/Prototypes/Entities/Clothing/Uniforms/base.yml

```diff
@@ -1,6 +1,17 @@
 - type: entity
   abstract: true
   parent: Clothing
+  id: ClothingWithSuitSensor
+  components:
+  - type: SuitSensor
+  - type: DeviceNetworkComponent
+    deviceNetId: Wireless
+  - type: WirelessNetworkConnection
+    range: 500
```

Comment:
Did you try out the range in game?

Reply:
Do you mean out of range? Yes, worked fine